Content/ArticleDataProvider.php

```diff
@@ -199,6 +207,29 @@ public function resolveDatasource($datasource, array $propertyParameter, array $
         return;
     }
 
+    /**
+     * @param array $propertyParameter
+     * @param array $options
+     *
+     * @return string|null
+     */
+    private function getWebspaceKey(array $propertyParameter, array $options)
+    {
+        if (array_key_exists('webspace', $propertyParameter)) {
```

Comment:
are you sure that `webspace` and `webspaceKey` in options exists?

Reply:
as i can see in `vendor/sulu/sulu/src/Sulu/Component/SmartContent/ContentType.php` there is only `webspaceKey`.